WORKSPACE

```diff
@@ -28,32 +28,34 @@ node_repositories()
 
 http_archive(
     name = "io_bazel_rules_closure",
-    sha256 = "59498e75805ad8767625729b433b9409f80d0ab985068d513f880fc1928eb39f",
-    strip_prefix = "rules_closure-0.3.0",
-    url = "http://bazel-mirror.storage.googleapis.com/github.com/bazelbuild/rules_closure/archive/0.3.0.tar.gz",
+    sha256 = "b29a8bc2cb10513c864cb1084d6f38613ef14a143797cea0af0f91cd385f5e8c",
+    strip_prefix = "rules_closure-0.8.0",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_closure/archive/0.8.0.tar.gz",
+        "https://github.com/bazelbuild/rules_closure/archive/0.8.0.tar.gz",
+    ],
 )
-
 load("@io_bazel_rules_closure//closure:defs.bzl", "closure_repositories")
-
 closure_repositories()
 
 # ================================================================
 
 git_repository(
     name = "io_bazel_rules_go",
     remote = "https://github.com/bazelbuild/rules_go.git",
-    tag = "0.2.0",
```

Comment:
Fixes this error:

```
ERROR: /tmpdir/external/io_bazel_rules_go/go/def.bzl:233:41: name 'set' is not defined
ERROR: error loading package '': Extension 'go/def.bzl' has errors
```
Can't use the latest rules_go because pubref/rules_protobuf still uses the deprecated go_repository:
https://github.com/pubref/rules_protobuf/blob/5cae42382b620aa1e347ecf30b3e92fd0d97998c/go/rules.bzl#L1

Instead we use the same version as pubref/rules_protobuf:
https://github.com/pubref/rules_protobuf/blob/5cae42382b620aa1e347ecf30b3e92fd0d97998c/WORKSPACE#L11